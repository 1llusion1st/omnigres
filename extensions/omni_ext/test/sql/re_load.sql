-- Pre-loaded value
SELECT omni_ext_test.alloc_shmem_global();

-- Change it
SELECT omni_ext_test.update_global_value('updated');

-- Re-load again
SELECT omni_ext.load('omni_ext_test');

-- Should not be rolled back to the initialized value
SELECT omni_ext_test.alloc_shmem_global();