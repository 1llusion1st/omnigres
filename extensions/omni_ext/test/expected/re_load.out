-- Pre-loaded value
SELECT omni_ext_test.alloc_shmem_global();
 alloc_shmem_global 
--------------------
 test
(1 row)

-- Change it
SELECT omni_ext_test.update_global_value('updated');
 update_global_value 
---------------------
 updated
(1 row)

-- Re-load again
SELECT omni_ext.load('omni_ext_test');
 load 
------
 0.1
(1 row)

-- Should not be rolled back to the initialized value
SELECT omni_ext_test.alloc_shmem_global();
 alloc_shmem_global 
--------------------
 updated
(1 row)

